src/Commands/Admin/SetTenantSite.cs

```diff
@@ -160,6 +160,9 @@ public class SetTenantSite : PnPAdminCmdlet
 
         [Parameter(Mandatory = false, ParameterSetName = ParameterSet_PROPERTIES)]
         public SiteUserInfoVisibilityPolicyValue OverrideBlockUserInfoVisibility { get; set; }
+                
+        [Parameter(Mandatory = false)]
+        public string InformationBarriersMode { get; set; }
```

Comment:
Shouldn't we enum this, if it can only be specific values as you indicate in the documentation proposal above? In that case people can easily tab through the options.